packages/date-picker/src/DatePicker/DatePicker.scss

```diff
@@ -33,15 +33,15 @@ $placeholder-opacity: 0.7;
   border-radius: $border-solid-border-radius;
   padding: 0 $button-base-padding-horizontal;
 
-  &:focus:not([disabled]),
-  &:hover:focus:not([disabled]),
+  &:global(.focus-visible):not([disabled]),
+  &:hover:global(.focus-visible):not([disabled]),
```

Comment:
I've never seen this before, for when it's focused and hovered at the same time? 🤯

Reply:
haha I removed this - I don't think its a thing

Reply:
Must of been added as overkill